Hakawai/Mentions/HKWMentionsPlugin.m

```diff
@@ -1677,6 +1677,15 @@ - (void)textViewDidChangeSelectionV2:(UITextView *)textView {
     if (range.length > 0) {
         return;
     }
+
+    // When cursor is placed within mention, highlight that mention.
```

Comment:
also could you put this in a separate function?

Reply:
Moved code in separate function.